.github/scripts/generateSpecMapping.js

```diff
@@ -0,0 +1,57 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+// Standalone specs can be added directly to the mapping. Any spec that requires displaying a version dropdown
+// will need to map its different versions to a separate Bump "branch". For example, a new resource version
+// for Atlas Admin API v2 will lead to a new entry in the array with its own Bump branch.
+const SPEC_MAPPING = [
+  {
+    doc: process.env.ATLAS_ADMIN_V1_DOC_ID,
+    file: 'openapi/v1-deprecated/v1.json',
+    branch: 'main',
+  },
+];
+
+function handleAdminAPIv2() {
```

Comment:
nit: can we please have unit tests?

Reply:
(ok to add in a follow-up PR)

Reply:
Okay, I can file a follow-up DOP ticket. Is there an ideal location within the repo to add this test? I didn't see a test directory for existing action scripts, but maybe I missed it 😅

Reply:
good point! I am not sure, in our case, we have scripts but they call some of the tools we built under the tools folder, which is where we have the most logic, if you think you'll build more on top of this script you can look into https://github.com/mongodb/openapi/tree/main/tools/spectral/ipa and create a tool folder for `docs` and have tests there, otherwise we can look into GH best practices on how to add coverage for the scripts!